### ISSUE TITLE
Problem with subroutes?

### ISSUE THREAD

**Issue by jurgenbosch** [spatie/laravel-menu#74]
Hi,

I'm building a blog category menu. 
So I have;
All - Travel - Sleeping - and so on

The route of All is /blog
The route of Travel is /blog/category/travel

If I'm visiting the /blog URL only the All category contains the class "Active". 
If I'm on /blog/category/travel is adds the class active to both All and Travel. 

I've tried using setActiveFromRequest as well as setActive(url()->full()) but both give the same outcome. 

If I change the slug of my named route (/blog) to /blogs the issue is gone. 
Basically it looks like the code is match if the nav item URL is **in** the URL instead of if it **is** the the URL.

**Comment by sebastiandedeyne** [comment]
You need to tell the class that `/blog` is a root url so it won't be flagged as active.

```
setActive(url()->full(), '/blog')
// Or
setActiveFromRequest('/blog')
```

In some cases you _want_ the top level url to be active too, so we can't be opinionated about this in the package.

Does this solve your issue?

**Comment by jurgenbosch** [comment]
Awesome! Solved. 
Couldn't made that out of the docs or codebase although I have seen the root option it was not clear to me that if was for this use. Might be something to improve in the docs and code base. 

PS. I love the package! It's awesome. Keep up the good work.
